File: medplum-java/medplum-java-model/src/main/java/com/fhir/Measure_Group.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableMeasure_Group.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface Measure_Group {
  Optional<CodeableConcept> code();

  Optional<List<Measure_Stratifier>> stratifier();

  Optional<List<Extension>> extension();

  Optional<List<Extension>> modifierExtension();

  Optional<List<Measure_Population>> population();

  Optional<String> description();

  Optional<String> id();

  static ImmutableMeasure_Group.Builder builder() {
    return ImmutableMeasure_Group.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/Measure_Group.ts
import { CodeableConcept } from "./CodeableConcept";
import { Extension } from "./Extension";
import { Measure_Population } from "./Measure_Population";
import { Measure_Stratifier } from "./Measure_Stratifier";

export interface Measure_Group {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    code: CodeableConcept | null | undefined;
    description: string | null | undefined;
    population: Measure_Population[] | null | undefined;
    stratifier: Measure_Stratifier[] | null | undefined;
}
